cluster-slots/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "solana-cluster-slots"
+description = "Solana Cluster Slots"
+documentation = "https://docs.rs/solana-cluster-slots"
+version = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+publish = false
```

Comment:
Any thoughts on the name of the crate? I'll need to reserve the name for publishing.